subsys/bluetooth/mesh/Kconfig

```diff
@@ -843,6 +843,10 @@ config BT_MESH_HEALTH_CLI_TIMEOUT
 
 endif # BT_MESH_HEALTH_CLI
 
+config BT_MESH_ALIGN_CHUNK_SIZE_TO_MAX_SEGMENT
```

Comment:
It should go under `BLOB models common configuration` section. Here: https://github.com/zephyrproject-rtos/zephyr/pull/76766/files#diff-007d797c63a5cbfd92a1922fc7fe3e40502988f252054ebeb6da1d77e5f25e87R944